### ISSUE TITLE
Send data through a proxy

### ISSUE THREAD

**Issue by Baklap4** [favll/pogom#202]
## Checklist
 - [ x] I have read the [FAQ](https://github.com/favll/pogom/blob/master/README.md#faq)
 - [ x] I have checked if this issue already exists (both open and closed issues)
 - [ x] I am using the most recent version of pogom from the master branch
 - [ x] If this is an installation issue, I am posting my OS, OS version, python version, pip version
 - [ x] I'm posting the complete log
 - [ ] I'm using a descriptive title for this issue

## My Issue
Is it possible to send data through a proxy so you can "lift" the softban?
...

**Comment by EiTeR94** [comment]
You mean like with a VPN?

**Comment by Baklap4** [comment]
My vps has a static ip instead of a dynamic one.
As for now i think my vps is permanent banned although i still can log in.
With a proxy my external ip would change to x.x.x.x where x can ben anywhere between 0 and 255.  Therefore "lifting" my soft/permaban

**Comment by alexanderhektor** [comment]
one proxy for all accounts doesn't help too long (already tested) - proxy support for each account/worker could massively help though.

**Comment by padinko** [comment]
PGOapi allow using proxy: https://github.com/keyphact/pgoapi/pull/66
can you add this?

**Comment by compuguy** [comment]
You may be able to set the environmental variable in Linux/Mac OS X, and the python library may use that?

**Comment by compuguy** [comment]
@alexanderhektor at this point any proxy support would be appreciated. It seems the server and libraries don't properly work with http_proxy and https_proxy variables.

**Comment by labaland** [comment]
IS it possible to host this map with vpn on? without ngrok? ports are open on router and vpn bou no one can connect to the site :/ only local

**Comment by compuguy** [comment]
I believe it may work with a proxy using environmental variables. I was able to pass them to my docker container. Just need to get a working proxy.                                                                                                                                                                        Sent via the BlackBerry Hub for Android                                                                                                                                                                                                                   From:notifications@github.comSent:August 15, 2016 1:57 PMTo:pogom@noreply.github.comReply-to:reply@reply.github.comCc:alex@compuguy1088.com; manual@noreply.github.comSubject:Re: [favll/pogom] Send data through a proxy (#202)  IS it possible to host this map with vpn on? without ngrok? ports are open on router and vpn bou no one can connect to the site :/ only local

—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/favll/pogom","title":"favll/pogom","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/favll/pogom"}},"updates":{"snippets":[{"icon":"PERSON","message":"@labaland in #202: IS it possible to host this map with vpn on? without ngrok? ports are open on router and vpn bou no one can connect to the site :/ only local"}],"action":{"name":"View Issue","url":"https://github.com/favll/pogom/issues/202#issuecomment-239876590"}}}

**Comment by xzyfer** [comment]
@padinko pgoapi added support for both proxy config and the standard HTTP_PROXY environment variables. The version of pgoapi of this project will need to update updated.

**Comment by compuguy** [comment]
@xzyfer I agree. What would make it easier if it was pulled in as a dependency, and not having its own local container.

**Comment by xzyfer** [comment]
@compuguy that's super weird. I expect it's because the author was afraid the repo was get C&D'd and disappear. In which case they could keep a private fork up to date as a fallback.

**Comment by compuguy** [comment]
So has any progress been done on allowing the use of proxies with pogom?
